tests/integration/test_auth_server_it.py

```diff
@@ -0,0 +1,1165 @@
+        assert created.description == TEST_DESC
+        assert created.audiences == TEST_AUDS
+        assert created.audiences[0] == TEST_AUDS[0]
+
+        # Update
+        UPDATED_DESC = f"{TEST_DESC}-Updated"
+        created.description = UPDATED_DESC
+        updated, _, err = await client.update_authorization_server(created.id,
+                                                                   created)
+        assert err is None
+        assert updated.description == UPDATED_DESC
+        assert updated.id == created.id
+
+        # Clean up
+        _, err = await client.deactivate_authorization_server(created.id)
+        assert err is None
+        _, err = await client.delete_authorization_server(created.id)
+        assert err is None
+
+    @pytest.mark.vcr()
+    @pytest.mark.asyncio
+    async def test_update_deactivate_delete_server(self, fs):
+        # Instantiate Mock Client
+        client = MockOktaClient()
+
+        # Create Auth Server
+        TEST_NAME = f"{TestAuthorizationServerResource.SDK_PREFIX}_test_abCeZ"
+        TEST_DESC = "Test Auth Server"
+        TEST_AUDS = ["api://default"]
+        auth_server_obj = models.AuthorizationServer({
+            "name": TEST_NAME,
+            "description": TEST_DESC,
+            "audiences": TEST_AUDS
+        })
+
+        created, _, err = await \
+            client.create_authorization_server(auth_server_obj)
+        assert err is None
+        assert isinstance(created, models.AuthorizationServer)
+        assert created.name == TEST_NAME
+        assert created.description == TEST_DESC
+        assert created.audiences == TEST_AUDS
+        assert created.audiences[0] == TEST_AUDS[0]
+
+        # Get Auth Server
+        auth_server, _, err = await client.get_authorization_server(created.id)
+        assert err is None
+        assert auth_server.id == created.id
+        assert auth_server.status == "ACTIVE"
+
+        # DeActivate Auth server
+        _, err = await client.deactivate_authorization_server(created.id)
+        assert err is None
+
+        # Get Auth Server
+        auth_server, _, err = await client.get_authorization_server(created.id)
+        assert err is None
+        assert auth_server.id == created.id
+        assert auth_server.status == "INACTIVE"
+
+        # Delete Auth server
+        _, err = await client.delete_authorization_server(created.id)
+        assert err is None
+
+        # Get Auth Server
+        auth_server, resp, err = await\
+            client.get_authorization_server(created.id)
+        assert err is not None
+        assert isinstance(err, OktaAPIError)
+        assert resp.get_status() == HTTPStatus.NOT_FOUND
+
+    @pytest.mark.vcr()
+    @pytest.mark.asyncio
+    async def test_activate_server(self, fs):
+        # Instantiate Mock Client
+        client = MockOktaClient()
+
+        # Create Auth Server
+        TEST_NAME = f"{TestAuthorizationServerResource.SDK_PREFIX}_test_abCeZ"
```

Comment:
Yes, this is for uniqueness. At first, I was generating the suffix dynamically, but since I'm using HTTP cassettes, the whole request (including the name of the Auth Server in the request) must match the cassette.

So, I resorted to unique endings per TC.